src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefFunctionalTest.java

```diff
@@ -321,8 +321,8 @@ public void shouldThrowForbiddenExceptionForNonCompliantRole() {
     @ToggleEnable(mapKey = CASEWORKER_FILE_UPLOAD, withFeature = true)
     public void shouldUploadXlsxFileSuccessfully() throws IOException {
         ExtractableResponse<Response> uploadCaseWorkerFileResponse =
-                uploadCaseWorkerFile("src/functionalTest/resources/CaseWorkerUserWithNoPassword.xlsx",
-                        200, REQUEST_COMPLETED_SUCCESSFULLY,
+                uploadCaseWorkerFile("src/functionalTest/resources/CaseWorkerUserXlsxWithNoPassword.xlsx",
+                        200, CaseWorkerConstants.REQUEST_COMPLETED_SUCCESSFULLY,
```

Comment:
Static import of `REQUEST_COMPLETED_SUCCESSFULLY` would be nicer

Reply:
sure, will change :)